### ISSUE TITLE
Unusual timestamps when downloading with CLI

### ISSUE THREAD

**Issue by FredrikMeyer** [jotta/jotta-cli-issues#176]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**

```
pi@fredriksraspberry:/mnt/uuidtest/jottacloud/Bilder/Bilder/2003-05 $ jotta-cli version
-------------------------------------------
jottad executable : /usr/bin/jottad
jottad appdata    : /home/pi/.jottad
jottad logfile    : /home/pi/.jottad/jottabackup.log
jottad version    : 0.14.58899
jotta-cli version : 0.14.58899
-------------------------------------------
OK
```

**Description of problem:**

The changed dates of pictures have strange dates:

```
pi@fredriksraspberry:/mnt/uuidtest/jottacloud/Bilder/Bilder/2003-05 $ ls -lh
total 14M
-rw-r--r-- 1 pi pi 349K Apr  8  1927 'Betenksom.jpg'
-rw-r--r-- 1 pi pi 627K Apr 17  1927 'Busker og fjell.jpg'
```

It says that they were edited in 1927...

**Expected:**

The correct date is around 2003. This shows correctly in the UI online.

**Comment by Kimbsen** [comment]
Thanks for the report. We'll get that fixed!

**Comment by FredrikMeyer** [comment]
Thank you!

Any updates on this?

**Comment by Kimbsen** [comment]
We just released 0.14.60453 ( unstable ) which fixes this.

We'll probably do a stable release with the same fix next week

**Comment by FredrikMeyer** [comment]
Thanks, this seems to be fixed now.
